Scripts/Services/SpeechToText/V1/SpeechToTextService.cs

```diff
@@ -52,13 +52,11 @@ public bool DisableSslVerification
         /// <summary>
         /// SpeechToTextService constructor.
         /// </summary>
-        
         public SpeechToTextService() : this(ConfigBasedAuthenticatorFactory.GetAuthenticator(serviceId)) {}
 
         /// <summary>
         /// SpeechToTextService constructor.
         /// </summary>
-        
```

Comment:
Was this regenerated?

Reply:
yeah, probably should not change it in this pr